features/support/world/language.rb

```diff
@@ -1,8 +1,13 @@
 module World
   module Language
-    def language_to_locale(language)
-      { 'English' => :en, 'Welsh' => :cy }[language]
+    def language
+      @language
+    end
+
+    def set_language(code)
```

Comment:
I would be slightly careful around using the term `locale`, as it has a much broader meaning in internationalisation than just translating languages. It also covers punctuation (`“` vs `<<`), currency, calendaring, sorting words (did you know digraphs are important when sorting Welsh?), and more.

Since you are using a language code (`en` is the language code for generic English, but `en-gb` is the locale for "English (Great Britain)", `code` is correct.

If you need another example, take Belgium. It has three official languages and several minority languages and dialects. You wouldn't set the `locale` to `nl` and claim it as "Belgish". You'd set it to `nl-be` and claim it as "Dutch (Belgium)". Which is different from `nl-nl` "Dutch (Netherlands)". But you can set the language to `nl`, generic Dutch, which many people in Belgium might prefer to read over French or German.